### ISSUE TITLE
Line height issue (Android vs iOS)

### ISSUE THREAD

**Issue by outaTiME** [hectahertz/react-native-typography#11]
Hi, I have the following style:

```js
class SettingsHomeScreen extends DrawerScreen {
  _onPress = () => {
    // pass
  };
  render() {
    return (
      <View style={styles.container}>
        <ScrollView>
          <View style={styles.card}>
            <View style={styles.card_container}>
              <Touchable onPress={this._onPress}>
                <View style={styles.group}>
                  <Text style={[ material.body2 ]}>Desconectarse</Text>
                </View>
              </Touchable>
            </View>
          </View>
        </ScrollView>
      </View>
    );
  }
```

I'm using Expo v23, I see the elements are being rendered different in iOS than Android, especially the vertical alignment of the texts. Are the line heights right? looks like the problem is related to them. 

Please look these screenshots:

iOS (great):

<img width="598" alt="screen shot 2017-12-13 at 16 53 27" src="https://user-images.githubusercontent.com/140472/33960102-68957e6e-e028-11e7-9b52-48471bba48c3.png">

Android (line height too high):

<img width="912" alt="screen shot 2017-12-13 at 16 53 36" src="https://user-images.githubusercontent.com/140472/33960133-811b4072-e028-11e7-963a-f81483cbc4fe.png">

**Comment by hectahertz** [comment]
Hi @outaTiME, thanks for the issue! 😄

This is the problem described here: https://github.com/hectahertz/react-native-typography#the-roboto-typeface-line-heights-are-not-100-accurate-to-the-material-styles-definition

This commit on master https://github.com/facebook/react-native/commit/3f1b0215060e4c27c286359cc90f3b2189956c4e fixes it, but it didn't get into a release yet ☹️

**Comment by hectahertz** [comment]
@outaTiME just a heads up, that commit landed in v0.52.0-rc.0

**Comment by outaTiME** [comment]
@hectahertz great, will be released soon !!!
